application/forms/Authentication/LoginForm.php

```diff
@@ -27,6 +27,13 @@ class LoginForm extends Form
      */
     const REDIRECT_URL = 'dashboard';
 
+    public static $defaultElementDecorators = [
+        ['ViewHelper', ['separator' => '']],
+        ['Help', []],
+        ['Errors', ['separator' => '']],
+        ['HtmlTag', ['tag' => 'div', 'class' => 'control-group']]
+    ];
```

Comment:
We can't do that. Due to two things:

a) `.control-group` is a structural class and not responsible for styling, hence should never be used for only styling reasons and here it is.
b) Changing the markup of the login form surely breaks third party themes in a big way. If it's only done to re-use our form styles, either make the form styles less specific or copy the style.

Reply:
I removed the .control-group elements and adjusted the styles with 86f8f0d.
✅